staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -111,6 +112,9 @@ func (o *RecommendedOptions) ApplyTo(config *server.RecommendedConfig) error {
 		if err := o.Traces.ApplyTo(config.Config.EgressSelector, &config.Config); err != nil {
 			return err
 		}
+		wrapper := traces.WrapperFor(config.TracerProvider)
+		o.Authentication.WithCustomRoundTripper(wrapper)
+		o.Authorization.WithCustomRoundTripper(wrapper)
```

Comment:
This needs to handle Authentication and/or Authorization being nil (or WithCustomRoundTripper needs to handle the receiver being nil by no-oping)

This also only applies to custom apiservers using `RecommendedOptions`, not kube-apiserver (it has ... more complex wiring setup ... done in `cmd/kube-apiserver/app/server.go#buildGenericConfig()`). How can we verify tracing is effective for the remote authn/authz options set up for kube-apiserver?

Reply:
You are correct.  I tested it, and this isn't tracing what I thought it was.